app/components/avo/index/resource_grid_component.html.erb

```diff
@@ -9,6 +9,6 @@
   </div>
 <% else %>
   <div class="bg-white rounded shadow-panel">
-    <%= helpers.empty_state resource_name: @resource.name.downcase.pluralize, related_name: params[:related_name], view_type: :grid %>
+    <%= helpers.empty_state resource_name: @resource.name(count: 0).downcase, related_name: params[:related_name], view_type: :grid %>
```

Comment:
Actually, let's create a new issue to track the `count: 0` problem, and just fix what's under the scope of this issue (the missing locale tags).

Reply:
I think this is the native translation package fix, we only can use the pluralize method on English.

> Using pluralized `one` instead `zero` on `no_item_found` and on `no_related_item_found` translation

 ```ruby
error_log:
        zero: "log de erro zero"
        one: "log de erro one"
        other: "logs de erro other"
```
![image](https://user-images.githubusercontent.com/69730720/188860997-78b9c158-e0ab-47fd-8dae-e9d2d8fc5b54.png)

As we can see from the original issue, we are using the name of the resource and pluralize it in order to get the empty state. Each language has it's own pluralization methods that's why i think using the zero count should work for every language in this case, we want to tell to user "0 related resource_name found". If `zero` count for that resource is not defined `one` will be used as default so it will never brakes.

https://guides.rubyonrails.org/i18n.html#pluralization

BTW i think this happens in every language with different words, not only on portughese
Example: In `avo.ro.yml` we have 
    `no_related_item_found: Nu s-au găsit %{item} asociate`
For `resource_name = "fruct"` we get the message `Nu s-au găsit fructe asociate`
For `resource_name = "jucator"` we get the message `Nu s-au găsit jucători asociate` and here we are stuck, can't translate it, with the cout implementation, we can change to:
 `no_related_item_found: Niciun %{item} asociat găsit`

```en:
  avo:
    resource_translations:
      fruct:
        zero: fruct
        one: fruct
        other: fructe
      comment:
        zero: jucător
        one: jucător
        other: jucători
```
Now we get always the correct message.
 Or, if the developer realy want can use the `i18n context` to give different translations for masculine or feminine words. My point is, we should not force this message to use always plural, sometimes it don't fit

Anyway, if you think this implementation is risky let me know and we can remove this part

Reply:
I understand now better what the issue is. Thanks for explaining.

I think the best solution would be to avoid using the phrase "no users found" and use something generic like "no items found" or "no related items found". This way we don't need to translate and figure out the pluralization in the same translation string.

What do you think?

Can we do that in this PR? So no `count: 0` argument?

Reply:
I agree, that way we reduce the unnecessary complexity here! Implementing it on next commit